lib/membrane_webrtc_plugin/sdp.ex

```diff
@@ -339,6 +358,9 @@ defmodule Membrane.WebRTC.SDP do
     Track.new(media_type, stream_id, opts)
   end
 
+  @doc """
+  Create simulcast track based on prototype track, rtp header extenstions, and EndpointBin extensions.
```

Comment:
```suggestion
  Creates simulcast track based on prototype track, rtp header extenstions, and EndpointBin extensions.
```  
Please checke other functions too

Also I am not sure if these are `EndpointBin extensions`. Aren't they RTP header extensions?

Reply:
Yeah, this is a little bit tricky to distinguish. Because the second parameter of the function is RTP header extensions, which we took from RTP packets, but the third parameter is modules which we pass as an option to EndpointBin and they are standardizing the way we handle RTP header extensions. So using the same words for describing two different parameters would be confusing. That's why I decide to use the name from the description of this opts in EndpointBin.